includes/theme.php

```diff
@@ -318,7 +320,7 @@ function ap_post_actions( $_post = null ) {
 	}
 
 	// Permanent delete link.
-	if ( ap_user_can_permanent_delete( $_post->ID ) ) {
+	if ( ap_user_can_permanent_delete( $_post->ID ) && ( ! ap_opt( 'deleting_question_with_answer' ) || ( ap_opt( 'deleting_question_with_answer' ) && empty( $answers ) ) ) ) {
```

Comment:
@mebishalnapit Can this be simplified to this:
```
if (ap_user_can_permanent_delete($_post->ID) && (!ap_opt('deleting_question_with_answer') || (empty($answers)))) {

```

Reply:
@rahularyan Modified the condition as requested.